Gemfile.lock

```diff
@@ -83,30 +83,31 @@ GEM
       rake (>= 10.4, < 14.0)
     apipie-rails (0.5.18)
       rails (>= 4.1)
-    arbre (1.2.1)
-      activesupport (>= 3.0.0)
-    ast (2.4.0)
+    arbre (1.3.0)
+      activesupport (>= 3.0.0, < 6.1)
+      ruby2_keywords (>= 0.0.2, < 1.0)
+    ast (2.4.1)
```

Comment:
### [ast](https://whitequark.github.io/ast/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.4.0 → 2.4.1

[_~~change-log~~, [source-code](https://github.com/whitequark/ast)_]





<details>
<summary>Commits</summary>

A change of **7** commits. See the full changes on [the compare page](https://github.com/whitequark/ast/compare/v2.4.0...v2.4.1).

These are the individual commits:
- (993d900) [Fix typo.](https://github.com/whitequark/ast/commit/993d900daaefc0558ce73d21d87f38a4a14ed016)
- (50ff345) [Only ship the necessary library files in the gem artifact](https://github.com/whitequark/ast/commit/50ff345ab7152bf513865b88e03664570942318b)
- (5ab62fa) [bump rake](https://github.com/whitequark/ast/commit/5ab62faa0c663e16c39979ea2b8d9cdf4148a243)
- (8beeda1) [update ci](https://github.com/whitequark/ast/commit/8beeda14dbb6ba9827e346213ca7dc47a8d7d0c4)
- (05d2321) [Removed unused dependencies from gemspec.](https://github.com/whitequark/ast/commit/05d232169ba36095bfb868157b01dd206b9c277b)
- (9a0bfdb) [Fixed AST::Node#updated to always return a copy. (#25)](https://github.com/whitequark/ast/commit/9a0bfdb9808527c982580cf5edd94a85af2b3805)
- (e07a4f6) [Bump version.](https://github.com/whitequark/ast/commit/e07a4f66e05ac7972643a8841e336d327ea78ae1)


</details>